02_Development/Display/constants.py

```diff
@@ -1,15 +1,25 @@
 # Constants for the display software for the
 # Tetra Biodistributed Ventillator Splitter
 
+PRESSURE_SENSOR_MUX_ADDRESS = 0x70
+FLOW_SENSOR_MUX_ADDRESS = 0x74
+
 NUMBER_OF_PATIENTS = 4
```

Comment:
Is their a way we can gauge this from the sensor values we get back? Or are we going to just show data for four patients regardless?

Reply:
probably a bigger question than just a constant to determine the array size, eh?